solidity/interfaces/IOngoingAirdrops.sol

```diff
@@ -0,0 +1,165 @@
+ * @notice This contract will enable multiple ongoing and updateable airdrops (called campaigns). Each of this campaigns
+ *         will have an updateable deadline, and will be able to hold airdrop information for multiple tokens.
+ *         Additionally, governor will be able to shutdown a campaign whenever it choses to. Since this is a pretty
+ *         permissioned contract, governor has full control over funds.
+ */
+
+interface IOngoingAirdrops {
+  /// @notice Token and amount information
+  /// @dev Struct that will hold an ERC20 token, and an amount.
+  struct TokenAmount {
+    IERC20 token;
+    uint256 amount;
+  }
+
+  /// @notice Thrown when a campaign is invalid, or zero bytes.
+  error InvalidCampaign();
+
+  /// @notice Thrown when a campaign does not exist or merkle root is invalid.
+  error InvalidMerkleRoot();
+
+  /// @notice Thrown when trying to update a campaign or claiming with invalid token amount.
+  error InvalidTokenAmount();
+
+  /// @notice Thrown when trying to do operations on a zero address.
+  error ZeroAddress();
+
+  /// @notice Thrown when validating an airdrop claim with the proof is invalid.
+  error InvalidProof();
+
+  /// @notice Thrown when trying to update a campaign's deadline to an invalid moment.
+  error InvalidDeadline();
+
+  /// @notice Thrown when trying to claim an expired campaign.
+  error ExpiredCampaing();
+
+  /// @notice Thrown when user tries to claim a campaign without having a new claim.
+  error AlreadyClaimed();
+
+  /// @notice Emitted when a campaign is updated
+  /// @param campaign Campaign name updated
+  /// @param root Merkle root that will be used to validate campaign claims
+  /// @param tokensAllocation Array of the sum of all tokens and amounts airdropped in the campaign
+  /// @param deadline Timestamp until when the campaign is claimable
+  event CampaignUpdated(bytes32 campaign, bytes32 root, TokenAmount[] tokensAllocation, uint32 deadline);
+
+  /// @notice Emitted when a balance a tranche gets closed
+  /// @param campaign Campaign being shut down
+  /// @param tokens Unclaimed tokens to take from campaign
+  /// @param amounts Amount of unclaimed tokens sent
+  /// @param recipient Address that will receive unclaimed tokens
+  event CampaignShuttedDown(bytes32 campaign, IERC20[] tokens, uint256[] amounts, address recipient);
+
+  /// @notice Emitted when a user claims a tranche.
+  /// @param campaign Campaign being claimed
+  /// @param claimee Address of the person claiming the airdrop
+  /// @param tokensAmount Tokens and amounts being used to get leaf
+  /// @param claimed Total amount of claimed tokens where token claimed = tokensAmount[index].token
+  /// @param recipient Address that will receive the tokens being claimed
+  event Claimed(bytes32 campaign, address claimee, TokenAmount[] tokensAmount, uint256[] claimed, address recipient);
```

Comment:
Since we can claim in behalf of others, would it make sense to add the account who triggered the claim?